src/br/com/vaasschool/model/QuestionType.java

```diff
@@ -3,7 +3,5 @@
 public enum QuestionType {
     SINGLE_ANSWER,
     MULTIPLE_ANSWERS,
-    TRUE,
-    FALSE
-
+    TRUE_FALSE
```

Comment:
Talvez colocar um `OR`?

```suggestion
    TRUE_OR_FALSE
```